scripts/gen_vimdoc.py

```diff
@@ -571,7 +573,7 @@ def ind(s):
     # text += (int(not space_preceding) * ' ')
 
     if n.nodeName == 'preformatted':
```

Comment:
After we get rid of all the `<pre>` we should throw an error here or some other way to signal the preferred markup.